pkg/controller/namespace/namespace_controller.go

```diff
@@ -34,10 +34,9 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/component-base/metrics/prometheus/ratelimiter"
+	"k8s.io/klog/v2"
 	"k8s.io/kubernetes/pkg/controller"
 	"k8s.io/kubernetes/pkg/controller/namespace/deletion"
-
-	"k8s.io/klog/v2"
```

Comment:
the intent was to keep imports from this repo together... k8s.io/klog is an external repo

Reply:
no, other imports also think it is k8s its own package, just as below
https://github.com/kubernetes/kubernetes/blob/3bd29bc53d5acb2ac794f4d150472f6a759f0e23/pkg/credentialprovider/aws/aws_credentials.go#L36-L40